### PR TITLE
Migration Issue fix for Google API v1beta to v1

### PR DESCRIPTION
**Pull request by bao1018**
couple of changes base on v2.4.2 this PR is going to fix below issue:#202
1. remove location property
2. set default region property value
3. add migration guide for v1beta to v1

### CODE REVIEW

**Comment by bao1018** [comment]
@medikoo thanks for your patience, this is the fully tested version, you can use your proposed process to publish the new release. thanks

- Revert #165
- Publish v2.4.3
- Take #165 again
- Publish as v3.0.0

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "serverless-google-cloudfunctions",
-  "version": "2.4.2",
+  "version": "3.0.0",
```
Comment (medikoo) [inline package.json]: This should done with dedicated release PR. Please get familiar with https://github.com/serverless/serverless-google-cloudfunctions/blob/master/RELEASE_PROCESS.md

**Comment by bao1018** [comment]
@zxhaaa6 can you address medikoo comments as soon as possible ?  thanks

**Comment by zxhaaa6** [comment]
@bao1018 sure, thanks

**Comment by zxhaaa6** [comment]
Hi @medikoo, could you please help to review? And after that, I would prepare v3.0.0 release PR. Many thanks!

**Comment by zxhaaa6** [comment]
Oh sorry, I will split Upgrade of googleapis (so drop of support for Node.js) into another PR. Thanks @medikoo

**Comment by zxhaaa6** [comment]
Hi @medikoo , could you please help to review? After that, I will do next step. Thanks!

**Comment by zxhaaa6** [comment]
@medikoo , a strange travis issue

**Comment by medikoo** [comment]
> @medikoo , a strange travis issue

Yes I saw, I've merged it as it didn't seem related to code changes

**Pull request merged**